app/cell.py

```diff
@@ -89,6 +90,12 @@ def mark(self, event=None):
             if self.field.number_of_bombs.get() == 0:
                 self.field.check()
 
+    def show(self, not_place_of_death):

```

Comment:
"Inverted" arguments are harder to understand and should be avoided. I suggest renaming to `place_of_death`. Don't forget to invert the value.